### PR TITLE
miscellaneous fixes

### PR DESCRIPTION
**Pull request by davidsmejia**
## Issue Number

n/a

## Purpose/Implementation Notes

* fixes the modalities on the samples table
* dynamically figures out which data is inside of the computed file

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Functional tests

n/a

## Checklist

- [X] Lint and unit tests pass locally with my changes

## Screenshots

n/a

### CODE REVIEW

File: api/scpca_portal/models/sample.py
Comment (arkid15r) [inline api/scpca_portal/models/sample.py:43]: Remove print.

File: client/src/config/downloadOptions.js
Comment (arkid15r) [inline client/src/config/downloadOptions.js:27]:
```suggestion
    header: 'Download Spatial Data',
```

File: api/scpca_portal/models/sample.py
Comment (arkid15r) [inline api/scpca_portal/models/sample.py:41]: I'd prefer to have this in properties section under static methods

File: client/src/components/Download.js
Comment (nozomione) [inline client/src/components/Download.js:114]: **Suggestion:** since we're now passing the `resource`, can we remove the line `computedFiles={resource.computed_files}` and destructure the `computed_files` value in `DownloadOptions` instead? 
(Or we'll keep it for the sake of readability?)

File: api/scpca_portal/models/sample.py
Comment (arkid15r) [inline api/scpca_portal/models/sample.py:58]: Not sure if that's the approach we normally use. Would setting it to N/A on the front-end for null modalities values look better?

**Review by nozomione — CHANGES_REQUESTED** [review]
The update changes look great. Please see my suggestion before merging. Thank you, David!

File: api/scpca_portal/models/sample.py
Comment (davidsmejia) [reply api/scpca_portal/models/sample.py:58]: its kinda a mix, we have some values as N/A and others are null. I think youre right though. Null might make more sense here

File: client/src/components/Download.js
Comment (davidsmejia) [reply client/src/components/Download.js]: yeah thats a good call, done!

File: api/scpca_portal/models/sample.py
Comment (arkid15r) [inline api/scpca_portal/models/sample.py:107]:
```suggestion

```

Comment (arkid15r) [reply api/scpca_portal/models/sample.py:107]: No return == return None

**Review by arkid15r — APPROVED** [review]
LGTM

**Review by nozomione — APPROVED** [review]
LGTM

File: api/scpca_portal/models/sample.py
Comment (davidsmejia) [reply api/scpca_portal/models/sample.py:107]: yeah I tend to prefer explicit returns in languages that are less strict about that, but I'm happy to defer to your preference

Comment (arkid15r) [reply api/scpca_portal/models/sample.py:107]: I see. For me it just looks cleaner this way.
Thank you!

**Pull request merged**
